extra/onnx.py

```diff
@@ -85,6 +85,15 @@ def get_run_onnx(onnx_model: ModelProto):
     "Softsign", "Asinh", "Acosh", "Atanh",  "Elu", "Celu", "Selu", "Xor", "Round", "Erf")
   }
 
+  # these values are expected to be python consts
+  required_input_python_consts: Dict[str, tuple[int, ...]] = {
```

Comment:
i like that these Tensor functions have correct types, but this mapping is weird and hard to maintain. will need a better way to organize these proto conversion stuff in the future